### PR TITLE
Add webapp debug command

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

The Makefile command expects this debug command to be defined

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "main": "src/index.js",
     "scripts": {
         "build": "webpack --mode=production --display-error-details --progress",
+        "debug": "webpack --mode=development --display-error-details --progress",
```
Comment (hanzei) [inline webapp/package.json:8]: The starter template uses a different command. Should we adopt what the starter template does here?

**Review by spirosoik — APPROVED** [review]
LGTM. let's just cross-check @hanzei comment

**Review by spirosoik — COMMENTED** [review]
In a separate note as the lint is failing should we we fix the issue now?

**Comment by mickmister** [comment]
@spirosoik Yes this lint issue has come up for all plugin projects suddenly. I've pinged Delivery about it here https://github.com/mattermost/mattermost-plugin-jira/pull/1005#issuecomment-1858233984

**Review by hanzei — APPROVED** [review]

**Comment by mickmister** [comment]
Closing in favor of https://github.com/mattermost/mattermost-plugin-gitlab/pull/428

**Pull request closed without merging**